examples/src/main/scala/App.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+import com.microsoft.hyperspace._
+import com.microsoft.hyperspace.Hyperspace
+import com.microsoft.hyperspace.index.{IndexConfig, IndexConstants}
+
+object App {
+  def main(args: Array[String]): Unit = {
+    // Create Spark session
+    val sparkConf = new SparkConf().setMaster("local[*]")
+    val spark = SparkSession.builder.config(sparkConf).getOrCreate()
+    spark.sparkContext.setLogLevel("ERROR")
+
+    // Sample department records
+    val departments = Seq(
+      (10, "Accounting", "New York"),
+      (20, "Research", "Dallas"),
+      (30, "Sales", "Chicago"),
+      (40, "Operations", "Boston"))
+
+    // Sample employee records
+    val employees = Seq(
```

Comment:
Can we move the data to JSON or CSV files so that we can re-use them for the C# sample app?

Reply:
I believe he's making it easy to submit the app to a cluster (as opposed to asking folks to copy csv files into storage accounts). What do you think? 

If we want to do this on the C# side, we can use `CreateDataFrame` [API](https://github.com/dotnet/spark/pull/338). Any thoughts?

Reply:
Sure, that's fine.

Reply:
Deleted my other comment regarding files.